src/picongpu/include/simulation_defines/unitless/physicalConstants.unitless

```diff
@@ -65,9 +65,9 @@ namespace picongpu
 
 
     /** Unit of mass */
-    BOOST_CONSTEXPR_OR_CONST float_64 UNIT_MASS = SI::ELECTRON_MASS_SI * double(particles::TYPICAL_NUM_PARTICLES_PER_MACROPARTICLE);
+    BOOST_CONSTEXPR_OR_CONST float_64 UNIT_MASS = SI::BASE_MASS_SI * double(particles::TYPICAL_NUM_PARTICLES_PER_MACROPARTICLE);
     /** Unit of charge */
-    BOOST_CONSTEXPR_OR_CONST float_64 UNIT_CHARGE = -1.0 * SI::ELECTRON_CHARGE_SI * double(particles::TYPICAL_NUM_PARTICLES_PER_MACROPARTICLE);
```

Comment:
typo in meaning

> we need

no need, one *can* skip.


Our `BASE_CHARGE_SI` (speciesConstants.param) *is* the `ELECTRON_CHARGE_SI` (physicalConstants.param) which is currently defined negative (so both are) and this does not change with this PR.

So in principle you are right, but if we change it we will cause "raw charge densities" of ions to be negative. That's counter-intuitive. Please keep the `-1.0` therefore, electrons will so far stay our species one.